bindings/go/main.go

```diff
@@ -54,62 +48,71 @@ var (
 // by the C library, into a proper Go error. If there is no error, this
 // will return nil.
 func makeErrorFromRet(ret C.C_KZG_RET) error {
-	err, ok := errorMap[ret]
-	if !ok {
-		panic(fmt.Sprintf("unexpected return value: %v", ret))
+	switch ret {
+	case C.C_KZG_BADARGS:
+		return ErrBadArgs
+	case C.C_KZG_ERROR:
+		return ErrError
+	case C.C_KZG_MALLOC:
+		return ErrMalloc
+	case C.C_KZG_OK:
+		return nil
 	}
-	return err
+	return fmt.Errorf("unexpected return value from c-library %v", ret)
```

Comment:
I removed the `panic` here. I think libraries should generally avoid `panic`, especially in cases where they already have a channel to signal error  - in this case just returning an error. The the caller can decide if it's panic-worthy or if it can clean up some files or whatever before exiting